<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetsWindow</class>
 <widget class="QDialog" name="AssetsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assets</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="_filterAll">
       <property name="toolTip">
        <string>Show all</string>
       </property>
       <property name="text">
        <string>All</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/filter_all.png</normaloff>:/images/filter_all.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="_filterWorking">
       <property name="toolTip">
        <string>Show working</string>
       </property>
       <property name="text">
        <string>Working</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/filter_working.png</normaloff>:/images/filter_working.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="_filterRevision">
       <property name="toolTip">
        <string>Show revisions</string>
       </property>
       <property name="text">
        <string>Revisions</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/filter_revisions.png</normaloff>:/images/filter_revisions.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="_filterOther">
       <property name="toolTip">
        <string>Show others</string>
       </property>
       <property name="text">
        <string>Other</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/filter_other.png</normaloff>:/images/filter_other.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="LineEdit" name="_searchBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="AssetsView" name="_view">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="_addButton">
           <property name="toolTip">
            <string>Add revisions...</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="_removeButton">
           <property name="toolTip">
            <string>Remove selected revisions</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="_gearButton">
           <property name="toolTip">
            <string>Actions...</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/images/gear.png</normaloff>:/images/gear.png</iconset>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>138</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="_refreshButton">
           <property name="toolTip">
            <string>Refresh</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="InspectorView" name="_inspector"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="_buttons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>..lineedit</header>
  </customwidget>
  <customwidget>
   <class>AssetsView</class>
   <extends>QTreeView</extends>
   <header>.view</header>
   <slots>
    <signal>selectionModified(QItemSelection,QItemSelection)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>InspectorView</class>
   <extends>QTableView</extends>
   <header>.inspector</header>
   <container>1</container>
   <slots>
    <slot>update(QItemSelection,QItemSelection)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_view</tabstop>
  <tabstop>_addButton</tabstop>
  <tabstop>_removeButton</tabstop>
  <tabstop>_gearButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_addButton</sender>
   <signal>clicked()</signal>
   <receiver>AssetsWindow</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_removeButton</sender>
   <signal>clicked()</signal>
   <receiver>AssetsWindow</receiver>
   <slot>removeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>AssetsWindow</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttons</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>AssetsWindow</receiver>
   <slot>buttonPressed(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>900</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_view</sender>
   <signal>selectionModified(QItemSelection,QItemSelection)</signal>
   <receiver>_inspector</receiver>
   <slot>update(QItemSelection,QItemSelection)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>removeSelected()</slot>
  <slot>refresh()</slot>
  <slot>buttonPressed(QAbstractButton*)</slot>
 </slots>
</ui>
